cli/flags.rs

```diff
@@ -791,6 +791,7 @@ TypeScript compiler cache: Subdirectory containing TS compiler output.",
     .arg(Arg::with_name("file").takes_value(true).required(false))
     .arg(reload_arg().requires("file"))
     .arg(ca_file_arg())
+    .arg(location_arg().conflicts_with("file"))
```

Comment:
This needs a `deno info`-specific help text. I think clap will allow overriding it here.

Reply:
What exactly should the help text contain?

Reply:
```
Show files used for web storage APIs etc. when running a script with '--location=<HREF>'
```

Reply:
Done